aten/src/ATen/core/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 # This file solely exists to let Caffe2 Android build get at the list
 # of core files without having to trundle through all of ATen's CMakeLists.txt
 
-FILE(GLOB ATen_CORE_HEADERS "*.h")
-FILE(GLOB ATen_CORE_SRCS "*.cpp")
-FILE(GLOB ATen_CORE_TEST_SRCS "*_test.cpp")
+FILE(GLOB_RECURSE ATen_CORE_HEADERS "*.h")
+FILE(GLOB_RECURSE ATen_CORE_SRCS "*.cpp")
+FILE(GLOB_RECURSE ATen_CORE_TEST_SRCS "*_test.cpp")
```

Comment:
Weren't you going to replace these globs with file lists? ;)

Reply:
it's still on my todo list for c10. Probably also makes sense for ATen/core, but that wasn't immediately on my radar.